cli/tests/activate.bats

```diff
@@ -3657,6 +3657,112 @@ EOF
 EOF
+)
+  assert_success
+  assert_output --regexp "project/.flox/run/.*.project.dev/bin.*default/.flox/run/.*.default.dev/bin"
+}
+
+@test "tcsh: repeat activation in .tcshrc doesn't break aliases" {
+  # We don't need an environment, but we do need wait_for_watchdogs to have a
+  # PROJECT_DIR to look for
+  project_setup_common
+
+  "$FLOX_BIN" init -d default
+  MANIFEST_CONTENTS_DEFAULT="$(cat << "EOF"
+    version = 1
+
+    [profile]
+    tcsh = """
+      alias default_alias echo "Hello default!";
+    """
+EOF
+  )"
+  echo "$MANIFEST_CONTENTS_DEFAULT" | "$FLOX_BIN" edit -d default -f -
+
+  "$FLOX_BIN" init -d project
+  MANIFEST_CONTENTS_PROJECT="$(cat << "EOF"
+    version = 1
+
+    [profile]
+    tcsh = """
+      alias project_alias echo "Hello project!";
+    """
+EOF
+  )"
+  echo "$MANIFEST_CONTENTS_PROJECT" | "$FLOX_BIN" edit -d project -f -
+
+  echo "eval \`$FLOX_BIN activate -d '$PROJECT_DIR/default'\`" > "$HOME/.tcshrc.extra"
+
+  # It would be better use tcsh -i to source .tcshrc,
+  # but that causes the tests to background because tcsh -i tries to open
+  # /dev/tty.
+  # Instead `flox activate -d default` manually to simulate sourcing
+  # .tcshrc
+
+  export TCSH="$(which tcsh)"
+  export EXPECT="$(which expect)"
+  run tcsh <(cat <<'EOF'
+    set alias_exists="`alias default_alias`"
+    if ("$alias_exists" == "") then
+      echo "default_alias not found"
+      exit 1
+    else
+    endif
+    # We can't double check the alias has been loaded because bash isn't
+    # interactive and discards it
+    setenv FLOX_SHELL "$TCSH"
+    setenv NO_COLOR 1
+    "$EXPECT" "$TESTS_DIR/activate/activate-command.exp" "$PROJECT_DIR/project" "which project_alias && which default_alias"
+EOF
+)
+  echo "$output" > /Users/zmitchell/src/flox/tcsh-profile-only/log.txt
+  assert_success
+  assert_output --partial "project_alias: 	 aliased to echo Hello project!"
+  assert_output --partial "default_alias: 	 aliased to echo Hello default!"
+}
+
+@test "tcsh: repeat activation in .tcshrc creates correct PATH ordering" {
+  # We don't need an environment, but we do need wait_for_watchdogs to have a
+  # PROJECT_DIR to look for
+  project_setup_common
+
+  "$FLOX_BIN" init -d default
+  MANIFEST_CONTENTS_DEFAULT="$(cat << "EOF"
+    version = 1
+EOF
+  )"
+  echo "$MANIFEST_CONTENTS_DEFAULT" | "$FLOX_BIN" edit -d default -f -
+
+  "$FLOX_BIN" init -d project
+  MANIFEST_CONTENTS_PROJECT="$(cat << "EOF"
+    version = 1
+EOF
+  )"
+  echo "$MANIFEST_CONTENTS_PROJECT" | "$FLOX_BIN" edit -d project -f -
+
+  # It would be better use bash -i to source .bashrc,
+  # but that causes the tests to background because bash -i tries to open
+  # /dev/tty.
+  # Instead `eval "$(flox activate -d default)"` manually to simulate sourcing
+  # .bashrc
+
+  echo "eval \`$FLOX_BIN activate -d '$PROJECT_DIR/default'\`" > "$HOME/.tcshrc.extra"
+
+
+  TCSH="$(which tcsh)" \
+  EXPECT="$(which expect)" \
+  run tcsh  <(cat <<'EOF'
```

Comment:
I think since you're not exporting TCSH and EXPECT you need
```suggestion
  run tcsh  <(cat <<EOF
```

And then some `\$` for things you want expanded later